strongbox-configuration/src/main/java/org/carlspring/strongbox/yaml/ObjectMapperSubtypes.java

```diff
@@ -0,0 +1,46 @@
+package org.carlspring.strongbox.yaml;
+
+import org.carlspring.strongbox.util.ServiceLoaderUtils;
+import org.carlspring.strongbox.yaml.repository.CustomRepositoryConfiguration;
+import org.carlspring.strongbox.yaml.repository.MutableCustomRepositoryConfiguration;
+import org.carlspring.strongbox.yaml.repository.remote.MutableRemoteRepositoryConfiguration;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class ObjectMapperSubtypes
```

Comment:
Cosmetic: I think this should be renamed to `ObjectMapperSubTypes`.

Reply:
Isn't `subtype` a single word ? :)

Reply:
Actually, you're right. Sorry.